superset/extensions/__init__.py

```diff
@@ -123,7 +124,7 @@ def init_app(self, app: Flask) -> None:
 cache_manager = CacheManager()
 celery_app = celery.Celery()
 csrf = CSRFProtect()
-db = SQLA()  # pylint: disable=disallowed-name
+db = get_sqla_class()()
```

Comment:
This will allow compatibility for sqlalchemy 1.x and sqlalchemy 2.x at the same time